README.md

```diff
@@ -1 +1,7 @@
 # hw01
+## Surrein Maniraju Homework 1
+Hi my name is *Surrein Maniraju*. I am a fourth year Sociology Major in the college. I am taking this class because there are many things I am interested in that require computing skills such as:
+
+* GIS
+* Network Analysis
+* Data Scraping from sites such as [Twitter](https://twitter.com/?lang=en)
```

Comment:
Great use of headers, emphasis, links and lists. Very clear and organized. Maybe try next time inserting a picture using
`![GitHub Logo](/images/logo.png)`